gems/smithy-client/lib/smithy-client.rb

```diff
@@ -40,13 +41,11 @@
 require_relative 'smithy-client/schema'
 require_relative 'smithy-client/shapes'
 require_relative 'smithy-client/structure'
-
-# endpoints
-require_relative 'smithy-client/endpoint_rules'
+require_relative 'smithy-client/param_validator'
 
 module Smithy
   # Base module for a generated Smithy gem.
   module Client
-    VERSION = File.read(File.expand_path('../VERSION', __dir__)).strip
+    VERSION = File.read(File.expand_path('../VERSION', __dir__.to_s)).strip
```

Comment:
Whats this change for?

Reply:
Apparently dir can be nil so I have been calling to_s on it always.